pkg/assembler/backends/ent/backend/slsa.go

```diff
@@ -140,15 +142,18 @@ func upsertSLSA(ctx context.Context, client *ent.Tx, subject model.ArtifactInput
 		UpdateNewValues().
```

Comment:
This has to be replaced with `DoNothing()`
```suggestion
		DoNothing().
```

Reply:
It skip the update and the test fail with this change

Reply:
ok